src/gmt_dcw.c

```diff
@@ -197,10 +204,24 @@ GMT_LOCAL int gmtdcw_find_country (char *code, struct GMT_DCW_COUNTRY *list, int
 	return (low);
 }
 
-GMT_LOCAL int gmtdcw_find_state (char *scode, char *ccode, struct GMT_DCW_STATE *slist, int ns) {
+GMT_LOCAL int gmtdcw_find_state (struct GMT_CTRL *GMT, char *scode, char *ccode, struct GMT_DCW_STATE *slist, int ns) {
 	/* Return state id given country and state codes using a linear search */
 	int i;
-
+	static struct GMT_DCW_CHINA_CODES gmtdcw_CN_codes[DCW_N_CHINA_PROVINCES] = {
```

Comment:
Need to check DCW version here. For GMT>=6.1.1 + DCW<=1.1.4, `CN.11` is converted to `CN.BJ`, but `CN.BJ` is not available in DCW<=1.1.4

Reply:
That is a good point!.  See new commit.